tests/rnaseq.nf.test

```diff
@@ -0,0 +1,37 @@
+nextflow_pipeline {
+
+    name "Build rnaseq references"
+    script "../main.nf"
+    tag "pipeline"
+
+    test("Run with profile test for rnaseq | --input assets/genomes/test/pipelines/default.yml") {
+
+        when {
+            params {
+                input  = "${projectDir}/assets/genomes/test/pipelines/default.yml"
+                outdir = "$outputDir"
+                tools  = 'bowtie,bowtie2,faidx,hisat2,hisat2_extractsplicesites,kallisto,make_transcripts_fasta,rsem_preparereference_genome,salmon,star'
```

Comment:
Should we break these out into their own params? I feel like it'll be easy to miss one with this long of a string.

Reply:
I can't remember if you can check these at the schema level like this? I think that's what happen in Demultiplex right?

Reply:
We can decide later at some point, but I'd rather have a single params for these than multiple